Formula/shelltestrunner.rb

```diff
@@ -3,7 +3,7 @@ class Shelltestrunner < Formula
   homepage "https://github.com/simonmichael/shelltestrunner"
   url "https://hackage.haskell.org/package/shelltestrunner-1.9/shelltestrunner-1.9.tar.gz"
   sha256 "cbc4358d447e32babe4572cda0d530c648cc4c67805f9f88002999c717feb3a8"
-  license "GPL-3.0"
+  license "GPL-3.0-or-later"
```

Comment:
The only indicator I see of `-or-later` is [here](https://github.com/simonmichael/shelltestrunner/blob/27dd4713b741a4ec2a01ffb0703c4bfbfdf7ce73/src/shelltest.hs#L4-L5). I think that's enough, but I'm not sure if that applies only to the `shelltest` and not `shelltestrunner`

Reply:
cc @simonmichael :)

Reply:
the license on the haskell.org index is actually weird among all these haskell formulae.

Reply:
Thank you, confirmed that both hledger and shelltestrunner are GPL v3 or later. shelltest and shelltestrunner are the same thing.

Reply:
Thanks for the confirmation, @simonmichael!

Reply:
yeah, thanks @simonmichael!!